### DISCUSSION TITLE
Resize with mstsc.exe supported?

### DISCUSSION THREAD

**Discussion by voltamund** [neutrinolabs/xrdp#2076] [Q&A] [answered]
Since 0.9.16 On-the-fly resolution change should be supported.
Should this also work when using mstsc.exe?
When I tried mstsc.exe it was not working. The youtube demos use the Microsoft Remote Desktop from the Microsoft Store.
I never use this app and when I tried it I could not connect at all - the app just closes completely when I try to connect.
With mstsc.exe I can connect without problems, it just does not resize.

**Comment by Nexarian** [comment, marked as answer]
I don't believe MSTSC supports resizing in the manner in which you speak. I think it's an inconsistency in the feature set between MSTSC and the Microsoft Remote Desktop app store client. MSTSC's resize functionality is very limited, and it essentially only extends to moving the window from one monitor with resolution A to a different monitor with resolution B and then maximizing the window.

To confirm this, I suggest trying to use MSTSC to resize a connection to another Windows desktop. I don't think you'll be able to other than in the manner described above. In order to fix this, we need to pressure Microsoft. I'm not sure why you're having issues with the Microsoft Store App. Make sure you're not using Network Level Authentication as that's not supported just yet.

That being said, there is another issue that even this limited resizing functionality doesn't fully work with MSTSC right now: https://github.com/neutrinolabs/xrdp/issues/1928

**Reply by matt335672** [reply]
That's an excellent summary from @Nexarian.

It may also be worth pointing out that the store app uses the Universal Windows Platform (UWP) and UWP has been [deprecated by Microsoft](https://github.com/microsoft/WindowsAppSDK/discussions/1615). What happens to this app in future is not at all clear at this point.

@voltamund - if you have a look in `/var/log/xrdp.log` you should get an idea as to whether the store app is not working. As @Nexarian says, NLA may well be the culprit here.
